services/database/user.ts

```diff
@@ -35,4 +35,17 @@ async function getUsers() {
     return users;
 }
 
-export { getUser, getUsers };
+/**
+ * Creates a new user corresponding to newUserData and returns the newly created user
+ * @param newUserData - data corresponding to the new user
+ * @returns - the newly created user
+ */
+async function createUser(newUserData) {
+    const user = await prisma.user.create({
+        // TODO: are created_at and updated_at passed in or auto generated?
+        data: newUserData,
```

Comment:
We also want to pass in information on the type of user it is. For example, if the user we are creating is a teacher, we want to also create a record in the teacher table.